src/lib.rs

```diff
@@ -597,6 +597,21 @@ where
     Ok(())
 }
 
+struct EscapeLinkTitle<'a>(&'a str);
+
+impl fmt::Display for EscapeLinkTitle<'_> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        for c in self.0.bytes() {
```

Comment:
Is there any specific reason to iterate `bytes()` instead of `chars()`?

Converting a byte to a char for writing most definitely breaks the encoding. My typical test for this is to try `你好👋` as title.

Reply:
Don't remember why I used `bytes()`, `chars()` is better